script-base.js

```diff
@@ -409,7 +558,7 @@ Generator.prototype.addSocialConnectionFactory = function (javaDir, importPackag
  *
  */
 Generator.prototype.addMainCSSStyle = function(style, comment) {
-    // Not Working this.useSass -> Undifined
+    // Not Working this.useSass -> Undifined --> this in the modules context would be jhipsterFunc
```

Comment:
either that or in your module you need to do `jhipsterFunc.useSass = jhipsterVar.useSass;` coz when you call a method from `jhipsterFunc` your this becomes `jhipsterFunc` and not the running generator context, better add it as param as you said above